pkg/oceancd/model/operator/install.go

```diff
@@ -0,0 +1,171 @@
+package operator
+
+import (
+	"encoding/json"
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+	"spot-oceancd-operator-commons/component_configs"
+)
+
+type InstallationConfig struct {
```

Comment:
change to `OperatorManagerConfig`

Reply:
we already have `OperatorManagerConfig`that stores argoRollouts and operator's configs that then converted into ConfigMap

the InstallationConfig also contains info about the operator manager, the config is used only for the installation of OM  

I suggest leave it as it is